src/components/forms/textChangeHandler.ts

```diff
@@ -0,0 +1,4 @@
+export const textChangeHandler = 
```

Comment:
this just wraps a handler that wants a plain text value, DRYing up the part that pulls `event.target.value` out of the original handler's arg, passing it to the wrapped handler as desired